util/test/management.go

```diff
@@ -0,0 +1,93 @@
+package test
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// Server management API server for testing
```

Comment:
Nice. I was expecting to find something like that somewhere in https://github.com/open-policy-agent/opa/tree/master/test/e2e, but couldn't. 😄

Reply:
Heh yeah maybe it could be moved there, or we could have a testing library as part of the SDK even. Leaving here for now.